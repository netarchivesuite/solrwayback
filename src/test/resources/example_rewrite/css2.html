<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
<head>
    <title>Specific problem</title>
    <meta charset="utf-8">
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link rel="stylesheet" media="screen" type="text/css" href="/rootpath/stylesheet/mainstyle.full.min.css?v=111218a">
      <link rel="stylesheet" media="print" type="text/css" href="/rootpath/stylesheet/print.min.css?v=021218_o2">
      <link rel="stylesheet" type="text/css" href="/rootpath/stylesheet/smoothness/jquery-ui-1.8.17.custom.css">
      <link rel="icon" href="/favicon.ico_o4" type="image/x-icon">
      <link rel="shortcut icon" href="/favicon.ico_o5" type="image/x-icon">
</head>
<body>
<p>Nothing here</p>
</body>
</html>"