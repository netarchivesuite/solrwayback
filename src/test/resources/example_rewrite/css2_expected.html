<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="da">
 <head>
  <title>Specific problem</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" media="screen" type="text/css" href="http://localhost:0000/solrwayback/services/notfound/">
  <link rel="stylesheet" media="print" type="text/css" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=2">
  <link rel="stylesheet" type="text/css" href="http://localhost:0000/solrwayback/services/notfound/">
  <link rel="icon" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=4" type="image/x-icon">
  <link rel="shortcut icon" href="http://localhost:0000/solrwayback/services/view?source_file_path=somesourcefile&amp;offset=5" type="image/x-icon">
 </head>
 <body>
  <p>Nothing here</p>  "
 </body>
</html>